src/components/PostPageDetail/index.js

```diff
@@ -34,14 +34,13 @@ import {fonts, normalize} from '../../utils/fonts';
 import {getCountCommentWithChildInDetailPage} from '../../utils/getstream';
 import ShareUtils from '../../utils/share';
 import StringConstant from '../../utils/string/StringConstant';
+import {COLORS} from '../../utils/theme';
 import BlockComponent from '../BlockComponent';
 import ContainerComment from '../Comments/ContainerComment';
 import WriteComment from '../Comments/WriteComment';
 import useWriteComment from '../Comments/hooks/useWriteComment';
 import LoadingWithoutModal from '../LoadingWithoutModal';
 import {Shimmer} from '../Shimmer/Shimmer';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and `withInteractionsManaged` has been removed. If there's no specific reason for the removal of `withInteractionsManaged`, it might affect the performance of the component by not managing interactions properly.
```diff
- import {withInteractionsManaged} from '../WithInteractionManaged';
- import {COLORS} from '../../utils/theme';
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->